pkg/cluster/internal/providers/docker/network.go

```diff
@@ -53,8 +51,5 @@ func ensureNetwork(name string, ipFamily config.ClusterIPFamily) error {
 	// TODO: ipv6 subnet should probably not be fixed
 	// Though maybe just require the user to handle this by creating the network
 	// as they desire before running kind ...
-	if ipFamily == config.IPv6Family {
-		return exec.Command("docker", "network", "create", "-d=bridge", "--ipv6", "--subnet=fc00:db8:2::/64", name).Run()
-	}
-	return exec.Command("docker", "network", "create", "-d=bridge", name).Run()
+	return exec.Command("docker", "network", "create", "-d=bridge", "--ipv6", "--subnet=fc00:db8:2::/64", name).Run()
```

Comment:
we can't create the same range in all networks, we should add some bytes hashing the name or something

Reply:
that is true, however this is not new to this PR, and currently there is a single caller with a fixed value. there's an existing TODO here, I'm limiting this PR to *just* resolving the aspect of "ipv4 clusters before ipv6 clusters => bad state"

Reply:
ups, I forgot that was hardcoded now XD

Reply:
what happens if the host does not have ipv6?
the command will fail?

Reply:
https://github.com/kubernetes-sigs/kind/pull/1526#issuecomment-621084660

Reply:
bad wording, not that IPv6, the host IPv6
`net.ipv6.conf.all.disable_ipv6 = 1`

Reply:
it still works but i'm not sure that setting is sufficient to test e.g. lack of modules

Reply:
```console
$ sudo sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 1

$ docker exec kind-control-plane ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: veth94b73da2@if3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 82:b9:38:07:5d:5b brd ff:ff:ff:ff:ff:ff link-netns cni-7e2721d4-185b-b646-b43f-41f797ddfa65
    inet 10.244.0.1/32 brd 10.244.0.1 scope global veth94b73da2
       valid_lft forever preferred_lft forever
3: vetheb4ad147@if3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 4e:c3:91:c2:4a:c0 brd ff:ff:ff:ff:ff:ff link-netns cni-ad9efa8f-e278-bed2-a72d-f634c116ede3
    inet 10.244.0.1/32 brd 10.244.0.1 scope global vetheb4ad147
       valid_lft forever preferred_lft forever
4: veth39517578@if3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:c6:7a:67:8c:b0 brd ff:ff:ff:ff:ff:ff link-netns cni-fbf3ca43-0688-1ff0-09ba-c90621b6407e
    inet 10.244.0.1/32 brd 10.244.0.1 scope global veth39517578
       valid_lft forever preferred_lft forever
17: eth0@if18: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:42:ac:12:00:02 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 172.18.0.2/16 brd 172.18.255.255 scope global eth0
       valid_lft forever preferred_lft forever
    inet6 fc00:db8:2::2/64 scope global nodad 
       valid_lft forever preferred_lft forever
    inet6 fe80::42:acff:fe12:2/64 scope link 
       valid_lft forever preferred_lft forever

```

Reply:
that's fair enough, I think that we can move forward with this :)

Reply:
i'm definitely nervous about that, maybe I can fire up a COS GKE node and see if it still works